Website/Providers/DataProviders/SqlDataProvider/08.00.00.05.SqlDataProvider

```diff
@@ -6,24 +6,17 @@ IF EXISTS (SELECT * FROM dbo.sysobjects WHERE id = object_id(N'{databaseOwner}[{
 GO
 
 CREATE PROCEDURE {databaseOwner}{objectQualifier}GetSchedule
- @Server varchar(150)
+ @Server nVarChar(150)
 AS
 BEGIN
-SELECT
-  S.*
-  , (SELECT max(S1.NextStart)
-   FROM {databaseOwner}{objectQualifier}ScheduleHistory S1
-   WHERE S1.ScheduleID = S.ScheduleID) as NextStart
- FROM {databaseOwner}{objectQualifier}Schedule S
- WHERE
-	(
-		@Server IS NULL OR 
-		S.Servers LIKE '%,' + @Server + ',%' OR
-		S.Servers LIKE @Server + ',%' OR
-		S.Servers LIKE '%,' + @Server OR
-		S.Servers = @Server OR
-		S.Servers IS NULL)
-	ORDER BY FriendlyName ASC
+    SELECT S.*, 
+           (SELECT max(S1.NextStart)
+             FROM  {databaseOwner}{objectQualifier}ScheduleHistory S1
+             WHERE S1.ScheduleID = S.ScheduleID) AS NextStart
+     FROM  {databaseOwner}{objectQualifier}Schedule S
+     WHERE IsNull(S.Servers, N'') = N''
```

Comment:
Pedro, this is not correct, as I am now adding a Komma before and at the end of the servers list, so each server is embraced by commas and will be found. there is no part-name match anymore and this option is as stable as the previous one.